scripts/pyenv_setup.sh

```diff
@@ -5,17 +5,9 @@
 # - This script assumes you're calling from the top directory of the repository
 set -eu
 
-# Check if a command is available
-require() {
-    command -v "$1" >/dev/null 2>&1
-}
-
-query_big_sur() {
-    if require sw_vers && sw_vers -productVersion | grep -E "11\." > /dev/null; then
-        return 0
-    fi
-    return 1
-}
+# XXX: Ideally, we should make this import relative rather than from where the Makefile invokes this
```

Comment:
Under bash, this is what I like to use:

```bash
HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd -P)"
```

Then you'd like, `source "${HERE}/lib.sh"`.

Reply:
This is a neat trick! Thank you!